modules/launcher/src/main/java/org/mule/module/launcher/MuleContainer.java

```diff
@@ -135,6 +136,10 @@ protected void init(String[] args) throws IllegalArgumentException
         {
             setStartupPropertiesFile(propertiesFile);
         }
+
+        JdbcDriverManagerInitializer jdbcDriverManagerInitializer = new JdbcDriverManagerInitializer();
```

Comment:
I don't get this: the jdbcDriverManagerInitializer is created on the container and uses the system classloader to initialize the drivers. How is this helping to fix the memory leak if the DB driver is inside the application classloader, which does not even exists at this point?